llvm/test/Feature/Repo/repo_hash_graph_contributions_1.ll

```diff
@@ -0,0 +1,38 @@
+; Test GO's information of Contributions: load and call instructions.
```

Comment:
As above: I’m not sure what this means. Maybe “ Test that we correctly generate contributions from load and call instructions”? Do both tests check for correctness of load instructions (which would seem redundant)?

Reply:
Updated the comments in cca1f2d.

> Do both tests check for correctness of load instructions (which would seem redundant)?

Both tests not only check for correctness of `load` instruction but also check there is no redundant contributions which added by `call` and `store` instructions. In the current master branch, when collecting the contributions and dependencies, both `store` and `call` instructions added additional `GV` into the contribution list. The issue is fixed in this PR. Therefore, tests are not redundant.